jujugui/templates/index.html

```diff
@@ -0,0 +1,215 @@
c0 .159-.026.503-.026.635v4.71h-.741v-1.905c-.053.08-.132.211-.318.317-.317.212-.688.238-.873.238-.926 0-1.746-.661-1.746-2.09 0-1.429.794-1.958 1.614-1.958.794 0 1.138.424 1.323.688l.026-.635zm-1.878.476c-1.032 0-1.059 1.085-1.059 1.376 0 .265.027.847.318 1.217.159.238.476.37.82.37.9 0 1.058-.687 1.058-1.454.027-.292.027-1.509-1.137-1.509zM20.082 290.544v2.487c0 .37.08.847.82.847.37 0 .688-.132.9-.423.158-.239.158-.53.158-.715v-2.17h.741v3.123c0 .08.027.502.027.714h-.741l-.027-.661c-.132.264-.396.688-1.217.688-.952 0-1.402-.556-1.402-1.297v-2.593zM24.58 292.608c-.027.794.317 1.376 1.111 1.376.476 0 .847-.238.9-.794h.714c-.026.185-.053.503-.318.794-.158.185-.555.502-1.322.502-1.217 0-1.8-.74-1.8-1.957 0-.768.16-1.376.768-1.8.37-.264.82-.29 1.058-.29 1.72 0 1.693 1.534 1.667 2.142H24.58zm2.01-.503c0-.37-.052-1.137-.952-1.137-.476 0-1.032.29-1.058 1.137zM28.549 288.877h.714v5.53h-.714zM32.306 294.434l-.556 1.428h-.74l.581-1.481-1.614-3.837h.82l1.218 3.043 1.217-3.043h.793zM37.862 292.238l-2.17-3.36h.953l1.64 2.725 1.667-2.726h.847l-2.17 3.387v2.143h-.767zM42.36 290.465c1.534 0 1.878 1.217 1.878 2.01 0 1.138-.635 2.064-1.958 2.064-1.27 0-1.905-.873-1.905-2.01.027-.953.477-2.064 1.985-2.064zm-.053 3.519c.555 0 1.164-.397 1.164-1.561 0-.953-.45-1.456-1.111-1.456-.45 0-1.19.292-1.19 1.509-.027.793.317 1.508 1.137 1.508zM46.064 290.544v2.487c0 .37.08.847.82.847.37 0 .688-.132.9-.423.158-.239.158-.53.158-.715v-2.17h.741v3.123c0 .08.027.502.027.714h-.741l-.026-.661c-.133.264-.397.688-1.218.688-.952 0-1.402-.556-1.402-1.297v-2.593zM50.139 291.338c0-.265-.027-.53-.027-.794h.74l.027.741c.133-.53.609-.714.926-.767.212-.027.37-.027.503-.027v.715c-.026-.027-.08-.027-.132-.027h-.212c-.847 0-1.058.503-1.058 1.164v2.038h-.741v-3.043zM53.525 293.27c0 .29.159.687.847.687.29 0 .529-.053.688-.238a.632.632 0 0 0 .132-.37c0-.37-.265-.45-.847-.582-.793-.186-1.481-.344-1.481-1.138 0-.714.608-1.164 1.508-1.164.476 0 1.164.132 1.376.608.106.212.106.397.106.556h-.741c.026-.582-.477-.635-.741-.635-.106 0-.794 0-.794.556 0 .344.344.45.556.502 1.058.265 1.296.318 1.534.53a.999.999 0 0 1 .291.714c0 .423-.238.794-.529.979-.344.211-.74.264-1.138.264-.793 0-1.508-.317-1.508-1.243h.741z" class="amst0" fill="#fff"/></g></svg><svg viewBox="0 0 30.823957 23.918333" id="card-mastercard"><g transform="matrix(.26458 0 0 .26458 -3.995 -3.995)"><path d="M36.3 105.3v-6c0-2.3-1.4-3.8-3.8-3.8-1.2 0-2.5.4-3.4 1.7-.7-1.1-1.7-1.7-3.2-1.7-1 0-2 .3-2.8 1.4v-1.2H21v9.6h2.1V100c0-1.7.9-2.5 2.3-2.5s2.1.9 2.1 2.5v5.3h2.1V100c0-1.7 1-2.5 2.3-2.5 1.4 0 2.1.9 2.1 2.5v5.3zm31.1-9.6H64v-2.9h-2.1v2.9H60v1.9h1.9v4.4c0 2.2.9 3.5 3.3 3.5.9 0 1.9-.3 2.6-.7l-.6-1.8c-.6.4-1.3.5-1.8.5-1 0-1.4-.6-1.4-1.6v-4.3h3.4zm17.8-.2c-1.2 0-2 .6-2.5 1.4v-1.2h-2.1v9.6h2.1v-5.4c0-1.6.7-2.5 2-2.5.4 0 .9.1 1.3.2l.6-2c-.4-.1-1-.1-1.4-.1zm-26.9 1c-1-.7-2.4-1-3.9-1-2.4 0-4 1.2-4 3.1 0 1.6 1.2 2.5 3.3 2.8l1 .1c1.1.2 1.7.5 1.7 1 0 .7-.8 1.2-2.2 1.2-1.4 0-2.5-.5-3.2-1l-1 1.6c1.1.8 2.6 1.2 4.1 1.2 2.8 0 4.4-1.3 4.4-3.1 0-1.7-1.3-2.6-3.3-2
```

Comment:
We need to figure out how to get this in here, I just pasted it in instead of importing the file contents.